tests/vm

```diff
@@ -16,6 +16,10 @@ waitInstanceBooted vm1
 echo "==> Test cleanly stopping a VM"
 lxc stop vm1
 
+echo "==> Ensure that we can't launch an aliased VM image as a container"
+lxc image alias create vmalias "$(lxc config get vm1 volatile.base_image)"
+! lxc launch vmalias || false
```

Comment:
As discussed, this breaks the assumption that VM images can be implicitly used to launch VMs without having to provide the `--vm` flag.